repos/fdbt-reference-data-service/src/retrievers/txc_unzipper/main.py

```diff
@@ -54,6 +55,12 @@ def lambda_handler(event, context):
             Namespace='FDBT/Reference-Data-Retrievers'
         )
     except Exception as e:
+        ssm_client.put_parameter(
```

Comment:
As this lambda is invoked for every file uploaded to S3 - a system wide issue will flood SSM api.